docs/5.x/reporting-api.md

```diff
@@ -65,7 +65,9 @@ _referrerName==Google,referrerName==Bing;country==IN_
     *   html
     *   rss (when **date** is a range for example date=last10 or date=previous15)
     *   original; to fetch the original PHP data structure. This is useful when you call the Piwik API [internally using the PHP code](/guides/querying-the-reporting-api)
-    
+
+*   **format_metrics**; defines the output format of the metrics' values.
+    *   By appending `&format_metrics=0`, the metrics' values are displayed as numerical values in the output instead of the default percent rate.
```

Comment:
Just for completeness: There are actually three possible values that can be provided here:
* `format_metrics=0`: metrics will not be formatted at all. Percent values are returned as rates - This might be best used if the numbers should be used for further processing
* `format_metrics=1`: All metrics are returned formatted (this also includes appending percent or money signs) - This might be best used if the data should be used for direct presentation
* `format_metrics=bc`: In that case only percent values are formatted, while all other values aren't. This behavior is deprecated and only still exists, as the Matomo UI expects it in some places. This is also the default behavior if nothing is provided. At some point this option will be removed and `format_metrics=0` will become the new default.

Reply:
@sgiehl Thank you. I will add this

Reply:
Apologies @gillnd . I guess I should have waited to merge your PR.

Reply:
@snake14 no worries at all, thanks for all your assistance. I've added sgiehl's comments in a [new pull request 801](https://github.com/matomo-org/developer-documentation/pull/802)